core/build.gradle

```diff
@@ -5,36 +5,38 @@ version = '0.15-SNAPSHOT'
 archivesBaseName = 'libdohj-core'
 
 dependencies {
-    compile 'org.bitcoinj:bitcoinj-core:0.15.9'
-    compile 'com.madgag.spongycastle:core:1.58.0.0'
+    implementation 'org.bouncycastle:bcprov-jdk15to18:1.68'
```

Comment:
That completely breaks on Android. Using BC is a huge pain on Android due to conflicts with the sytsem supplied version.

Reply:
Does it still break? bitcoinj moved over to Bouncycastle ( https://github.com/bitcoinj/bitcoinj/blob/master/core/build.gradle#L11) so I presumed Android had unscrewed things.

If not, I'd have to make a new spongycastle as 1.58 was the last one.

Reply:
https://github.com/rtyley/spongycastle/issues/34 is relevant